src/main/webapp/app/components/oncokbTable/OncoKBTable.tsx

```diff
@@ -47,20 +48,37 @@ export default class OncoKBTable<T> extends React.Component<
     });
   }
 
+  SearchBar = () => {
```

Comment:
We would not recommend having a component within a class component. You could consider moving the logic to a functional component.